archlinuxcn/cabal-fmt/lilac.yml

```diff
@@ -0,0 +1,12 @@
+ maintainers:
+  - github: berberman
+
+update_on:
+  - source: archpkg
+    archpkg: haskell-optparse-applicative
+  - source: archpkg
+    archpkg: haskell-tasty
+  - source: archpkg
+    archpkg: haskell-tasty-golden
+  - source: hackage
+    hackage: cabal-fmt
```

Comment:
Shouldn't this come first (so that `_G.newver` returns the version of it)?

Reply:
Sorry, I thought the order would make no difference...